app/build.gradle

```diff
@@ -61,7 +61,7 @@ dependencies {
 
   //networking
   compile 'com.squareup.retrofit2:retrofit:2.1.0'
-  compile 'com.squareup.retrofit2:converter-gson:2.0.0'
+  compile 'com.squareup.retrofit2:converter-gson:2.1.0'
```

Comment:
Good catch! I understand this as the textbook case for using variables to manage dependency version numbers. What do you think of...

```
project.ext.set("retrofitVersion", "2.1.0")

dependencies {
  ...
  compile "com.squareup.retrofit2:retrofit:${retrofitVersion}"
  compile "com.squareup.retrofit2:converter-gson:${retrofitVersion}"
  ...
}
```